src/index.html

```diff
@@ -7,6 +7,41 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <main class="main-content">
+      <div class="calendar calendar--start-day-fri calendar--month-length-29"
```

Comment:
```suggestion
      <div class="calendar calendar--fri calendar--29"
```
Можно вот так записывать